include/lbann/layers/learning/channelwise_scale_bias.hpp

```diff
@@ -0,0 +1,144 @@
+////////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2014-2019, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory.
+// Written by the LBANN Research Team (B. Van Essen, et al.) listed in
+// the CONTRIBUTORS file. <lbann-dev@llnl.gov>
+//
+// LLNL-CODE-697807.
+// All rights reserved.
+//
+// This file is part of LBANN: Livermore Big Artificial Neural Network
+// Toolkit. For details, see http://software.llnl.gov/LBANN or
+// https://github.com/LLNL/LBANN.
+//
+// Licensed under the Apache License, Version 2.0 (the "Licensee"); you
+// may not use this file except in compliance with the License.  You may
+// obtain a copy of the License at:
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the license.
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef LBANN_LAYER_LEARNING_CHANNELWISE_SCALE_BIAS_HPP_INCLUDED
+#define LBANN_LAYER_LEARNING_CHANNELWISE_SCALE_BIAS_HPP_INCLUDED
+
+#include "lbann/layers/layer.hpp"
+#include "lbann/models/model.hpp"
+#include "lbann/utils/exception.hpp"
+
+namespace lbann {
+
+/** @brief Apply scale and bias to tensor channels.
+ *
+ *  The input tensor is sliced along the first tensor dimension (the
+ *  "channel" dimension, assuming image data in CHW format) and scale
+ *  and bias terms are applied independently to each slice. More
+ *  precisely, given input and output tensors
+ *  @f$ X,Y\in\mathbb{R}^{d_1\times\cdots\times d_n} @f$
+ *  and scale and bias vectors @f$ a,b\in\mathbb{R}^{d_1} @f$:
+ *  @f[
+ *    Y_{i,j,\cdots} = a_i X_{i,j,\cdots} + b_i
+ *  @f]
```

Comment:
It may be helpful to users to explicitly note the relationship to batchnorm here.